### ISSUE TITLE
Installed version 1.0.8 - We are so sorry, but something went terribly wrong when starting or running this add-on. 

### ISSUE THREAD

**Issue by MelvinRC** [elad-bar/DahuaVTO2MQTT#16]
Now installed 1.0.8. so now a thing for this repo? Still errors :-(

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing...
Add-on: DahuaVTO2MQTT
Listens to events from Dahua VTO unit and publishes them via MQTT Message
Add-on version: 1.0.8
There is an update available for this add-on!
Latest add-on version: null
Please consider upgrading as soon as possible.
parse error: Expected string key before ':' at line 1, column 4
[17:51:29] ERROR: Unknown HTTP error occured
System: (armv7 / raspberrypi4)
Home Assistant Core: 0.116.4
Home Assistant Supervisor: 2020.10.0
Please, share the above information when looking for help
or support in, e.g., GitHub, forums or the Discord chat.
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing...
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[17:51:31] INFO: Staring Dahua to MQTT
2020-10-31 16:51:31 <*** Dahua VTO Event Listener START ***>
PHP Fatal error: Uncaught Error: Call to undefined function curl_init() in /app/DahuaVTO.php:106
Stack trace:
#0 /app/DahuaVTO.php(91): DahuaVTOAPI->SetDeviceDetails()
#1 /app/DahuaVTO.php(179): DahuaVTOAPI->DahuaVTOAPI()
#2 /app/DahuaVTO.php(9): DahuaVTOEventListener->DahuaVTOEventListener()
#3 {main}
thrown in /app/DahuaVTO.php on line 106
[cmd] /bin/ash exited 255
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing...
            Oops! Something went wrong.
We are so sorry, but something went terribly wrong when
starting or running this add-on.

Be sure to check the log above, line by line, for hints.
[cont-finish.d] 99-message.sh: exited 0.
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.

**Comment by elad-bar** [comment]
The error is not related to this repo because it caused by the installation part of the addons which lacks of cURL,
Anyway, I will add a safe block to ignore that error so it will work for you,
Thanks for the update

**Comment by elad-bar** [comment]
And you have nothing to be sorry 😄

**Comment by MelvinRC** [comment]
> And you have nothing to be sorry 😄

Thanks, subject is a part of the error =)

[cont-finish.d] 99-message.sh: executing...
Oops! Something went wrong.
We are so sorry, but something went terribly wrong when
starting or running this add-on

Thanks for all the work.

**Comment by MelvinRC** [comment]
Can I do anything to get the current add-on up and running?

**Comment by dmitrychepel** [comment]
revert ver 1.0.5 from snapshot of ha

**Comment by cibernox** [comment]
I'm getting a similar message when starting the addon: 

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Add-on: DahuaVTO2MQTT
 Listens to events from Dahua VTO unit and publishes them via MQTT Message
-----------------------------------------------------------
 Add-on version: 1.0.9
 You are running the latest version of this add-on.
parse error: Expected string key before ':' at line 1, column 4
[21:02:02] ERROR: Unknown HTTP error occured
 System:   (armv7 / raspberrypi4)
 Home Assistant Core: 0.118.5
 Home Assistant Supervisor: 2020.12.6
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[21:02:04] INFO: @@@@@@@@@@@@@@@@
/usr/bin/curl
[21:02:04] INFO: @@@@@@@@@@@@@@@@
[21:02:04] INFO: FFS
[21:02:04] INFO: Staring Dahua to MQTT
2020-12-13 21:02:04	DEBUG	DahuaVTOAPI::SetDeviceDetails Device Type: VTO2202F
2020-12-13 21:02:04	DEBUG	DahuaVTOAPI::SetDeviceDetails Serial Number: 6H003ACPAJFB40F
PHP Notice:  Undefined offset: 1 in /app/DahuaVTO.php on line 49
PHP Fatal error:  Uncaught Error: Class 'Mosquitto\Client' not found in /app/DahuaVTO.php:126
Stack trace:
#0 /app/DahuaVTO.php(4): DahuaVTOEventListener->DahuaVTOEventListener()
#1 {main}
  thrown in /app/DahuaVTO.php on line 126
[cmd] /bin/ash exited 255
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
-----------------------------------------------------------
                Oops! Something went wrong.

 We are so sorry, but something went terribly wrong when
 starting or running this add-on.
 
 Be sure to check the log above, line by line, for hints.
-----------------------------------------------------------
```
Any clue of what might be causing it? This is home assistant installed on a RPi 4

**Comment by MelvinRC** [comment]
I don't think the issue is the same, my problem was fixed with version 1.0.9.
Is your Mosquitto config setup correctly in the add-on?

**Comment by cibernox** [comment]
@MelvinRC on a second look seems that the problem is not related, but I'm not the only one getting it: https://github.com/elad-bar/Hassio-addons/issues/7

**Comment by mayobuz** [comment]
> I don't think the issue is the same, my problem was fixed with version 1.0.9.
> Is your Mosquitto config setup correctly in the add-on?

Official mosquitto broker addon was installed properly, mqtt works with tasmotas like a charm. However I get same error like cibernox.

**Comment by elad-bar** [comment]
This repo is for the docker version not addon,
Anyway, fixed both, should work now, pls let me know if there is an issue

Thanks
